packages/flutter_nekoton_bridge/example/pubspec.yaml

```diff
@@ -44,6 +44,8 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
   freezed_annotation: ^2.2.0
+  #  Used for tests
+  http: 0.13.5
```

Comment:
Only for tests? Then in should be in dev_dependencies imho.

Reply:
-